cob_bringup/robots/cob4-7.xml

```diff
@@ -244,9 +244,8 @@
 			</include>
 		</group>
 
-		<include file="$(find cob_bringup)/drivers/mimic.launch">
-			<arg unless="$(arg sim)" name="rotation" value="270"/>
-		</include>
+		<include file="$(find cob_bringup)/drivers/mimic.launch"/>
```

Comment:
did you change the head pc display configuration? why is the rotation not needed?